h/accounts/test/views_test.py

```diff
@@ -306,13 +306,9 @@ def test_login_ajax_raises_ValidationFailure_on_ValidationFailure():
     assert exc_info.value.error.asdict() == {'password': 'too short'}
 
 
-@pytest.mark.usefixtures('routes_mapper')
+@pytest.mark.usefixtures('routes_mapper', 'session')
 def test_logout_ajax_returns_status_okay():
-    request = DummyRequest()
-
-    result = AjaxAuthController(request).logout()
-
-    assert result['status'] == 'okay'
+    AjaxAuthController(DummyRequest()).logout()['status'] == 'okay'
```

Comment:
Also the three line layout is deliberate. The three sections are the canonical [Arrange, Act, Assert](http://c2.com/cgi/wiki?ArrangeActAssert), so removing them isn't necessarily helpful.